conda-recipe/meta.yaml

```diff
@@ -7,7 +7,7 @@ build:
 
   script:
     - cd $RECIPE_DIR/..
-    - $PYTHON setup.py install
+    - LDFLAGS="-shared" FFLAGS=-fPIC $PYTHON setup.py install # [not win]
```

Comment:
Don't know why, but it would not "conda build" build without, I tried both on OSX and Linux. Tomorrow I can fire up a virtualbox with windows, and see what conda build does there